lib/color.h

```diff
@@ -34,9 +34,9 @@ enum class Color {
     FgMagenta  = 35,
     FgDefault  = 39
 };
-CPPCHECKLIB std::ostream& operator<<(std::ostream& os, const Color& c);
+CPPCHECKLIB std::ostream& operator<<(std::ostream& os, Color c);
 
-CPPCHECKLIB std::string toString(const Color& c);
+CPPCHECKLIB std::string toString(Color c);
```

Comment:
well.. if I think it's better that the argument constness in the prototype matches the definition. what do we do. can't you turn off this clang noise or stop using clang.

Reply:
It has nothing to do with Clang...

Looking at these changes having an unnecessary `const` in the header be interpreted as code where you might have forgotten to add the reference to the parameter. But that assumption falls apart in case where the body in inline. I will check if those are correctly detected as unnecessary copies though.

Reply:
Done.